env/build_noaacloud_gnu.env

```diff
@@ -0,0 +1,44 @@
+#environment for building on NOAA-Cloud
+module purge
+module use /contrib/Mark.Potts/spack/share/spack/modules/linux-centos7-haswell
+module use /contrib/Mark.Potts/spack/share/spack/modules/linux-amzn2-skylake_avx512
+module use /contrib/EPIC/hpc-modules/modulefiles/stack/
+module load gcc/9.3.0
+module load openmpi
+module load hpc 
+module load hpc-gcc 
+module load hpc-openmpi
+module use /apps/modules/modulefiles 
+module load rocoto
+module load netcdf/4.7.4 
```

Comment:
@mark-a-potts Can you make your env files consistent with the rest by making use of the "srw_common" module rather than loading each individually ?

Reply:
Pinging @mark-a-potts on this change.  We can merge after this is complete.  Thanks!

Reply:
@mark-a-potts

Reply:
Still waiting to get the modules installed on the cloud lined up with srw_common. Hopefully, that will be finished in the next few days.

Reply:
Thanks, @mark-a-potts!